spec/functions/number.md

```diff
@@ -632,6 +632,8 @@ digit-size-option = "0" / (("1"-"9") [DIGIT])
 If the value of a _digit size option_ does not evaluate as a non-negative integer,
 or if the value exceeds any implementation-defined and option-specific upper or lower limit,
 a _Bad Option_ error is emitted.
+If the value exceeds an implementation-defined upper or lower limit, 
+the implementation MAY replace it with an implementation-defined value.
```

Comment:
To resolve the ambiguity noted by @sffc in https://github.com/unicode-org/message-format-wg/pull/1070#discussion_r2043120996:
```suggestion
a _Bad Option_ error is emitted and
the implementation MAY ignore the option value or
replace it with an implementation-defined value.
```

Reply:
One part of the change that I'm proposing here is allowing an implementation to behave the same way for a digit size option ending up with either `-1` or `0` as values for an option that has an implementation-defined minimum of `1`. Is there a reason why this should not be allowed?

Reply:
There is no reason that should not be allowed. I think my proposal allows that (the implementation can ignore the option **_or_** replace it)?